gems/smithy/spec/spec_helper.rb

```diff
@@ -73,6 +60,38 @@ def create_plan(modules, model, type, options)
       }
       Smithy::Plan.new(model, type, plan_options)
     end
+
+    ## end temporary
+
+    def generate_client_gem(options = {})
+      plan = ClientHelper.generate(:client, options)
+      RbsSpyTest.setup(modules, sdk_dir) if ENV.fetch('SMITHY_RUBY_RBS_TEST', false)
```

Comment:
Does this work? Where does `modules` get defined?

Reply:
It doesn't work. Sloppy editing.